pkg/cmd/login.go

```diff
@@ -43,6 +43,15 @@ var loginCmd = &cobra.Command{
 		copyAuthCode := !utils.GetBoolFlag(cmd, "no-copy")
 		hostname, _ := os.Hostname()
 
+		// warn user if scope already contains a token
+		if prevConfig.Token.Value != "" {
+			prevScope, err1 := filepath.Abs(prevConfig.Token.Scope)
+			newScope, err2 := filepath.Abs(scope)
+			if err1 == nil && err2 == nil && prevScope == newScope {
+				utils.LogWarning("This scope already contains a token and will be overridden.")
```

Comment:
can we add a prefix to the message like: `IMPORTANT: This scope already contains a token and will be overridden.`?

Reply:
It already has a yellow `Warning` prefix.

Reply:
You are right.

Reply:
Merge and ship